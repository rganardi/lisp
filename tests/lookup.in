(define 0 (quote zero))
0
(define 1 (quote one))
1
0
(define 0 1)
0
(define 1 (quote a))
0
1
