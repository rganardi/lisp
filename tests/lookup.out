parse_eval: unspecified
zero
parse_eval: unspecified
one
zero
parse_eval: unspecified
one
parse_eval: unspecified
one
a
